docs/documentation/soc-and-feeds.md

```diff
@@ -5,196 +5,411 @@ slug: /soc-and-feeds
 sidebar_label: SOC and Feeds
 ---
 
-## 🚧 Under Construction 🚧
-:::caution 🚧 This page is under construction
 
-This section is still being worked on. Check back soon for updates!
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
 
-:::
+Swarm provides the ability to store content in content-addressed [chunks](https://docs.ethswarm.org/docs/concepts/DISC/#content-addressed-chunks-and-single-owner-chunks) (CAC) whose addresses are derived from the chunk data, or Single Owner Chunks (SOC) whose addresses are derived from the uploader's own public key and chosen identifier. With single owner chunks, a user can assign arbitrary data to an address and attest chunk integrity with their digital signature.
```

Comment:
> Single Owner Chunks (SOC) whose addresses are derived from the uploader's own public key and chosen identifier

To be more precise, it is derived from uploader **address** and chosen identifier